orchagent/pfcwdorch.cpp

```diff
@@ -432,6 +432,17 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::unregisterFromWdDb(const Port& po
 
         // Unregister in syncd
         m_flexCounterTable->del(key);
+
+        auto entry = m_entryMap.find(queueId);
+        if (entry != m_entryMap.end())
+        {
+            if (entry->second.handler != nullptr)
+            {
+                entry->second.handler->commitCounters();
+                entry->second.handler = nullptr;
```

Comment:
I think this is not required. Entry is erased in code below.

Reply:
Please be aware even if entry is erased counter are not commited

Reply:
If there are existing action handler， commitcounter are needed when entry are deleted.

Reply:
Yes, I mean that line setting it to nullptr is not required
I agree with commitCounters